src/guessCarrier.coffee

```diff
@@ -150,6 +150,7 @@ CARRIERS = [
   {name: 'dhlgm', regex: /^94748\d{17}$/, confirm: _confirmUsps22}
   {name: 'dhlgm', regex: /^93612\d{17}$/, confirm: _confirmUsps22}
   {name: 'dhlgm', regex: /^GM\d{16}/}
+  {name: 'dhl', regex: /^\d{10}/}
```

Comment:
Thanks for the PR @kas84.  However, I'm concerned about false positives here.  You're basically saying that any number with 10 or more digits is a DHL tracking number?  That doesn't sound very convincing.  While I agree many other regular expressions are 100% fool proof, and there's some possibility of false positives.  But your `^\d{10}` regex will trigger for a lot more false positives than any other regex.  Do you agree?